src/operator/regression_output.cc

```diff
@@ -34,24 +34,53 @@ Operator *RegressionOutputProp<type>::CreateOperator(Context ctx) const {
 DMLC_REGISTER_PARAMETER(RegressionOutputParam);
 
 MXNET_REGISTER_OP_PROPERTY(LinearRegressionOutput, RegressionOutputProp<reg_enum::kLinear>)
-.describe("Use linear regression for final output, this is used on final output of a net.")
-.add_argument("data", "NDArray-or-Symbol", "Input data to function.")
-.add_argument("label", "NDArray-or-Symbol", "Input label to function.")
+.describe(R"code(Use the LinearRegressionOutput as the final output layer of a net.
+
+LinearRegressionOutput is used to compute a loss for training. It enables the new network to optimize for squared loss.
```

Comment:
The like how you provided a math formula for MAERegressionOutput. Can you ensure all these Output operators follow the same template?